distribution/docs/src/main/resources/content/_running/command-scheduler.adoc

```diff
@@ -1,15 +1,13 @@
 :title: Command Scheduler
-:type: maintaining
+:type: maintainingIntro
 :status: published
 :summary: Introduction to the Command Scheduler.
 :order: 01
 
-==== Command Scheduler
+== Command Scheduler
```

Comment:
I think this would fit better under `Console Commands` instead of under `Maintaining`.

Reply:
I can get behind that. Moved.